### ISSUE TITLE
Error applying NormalizeVideo transform to ImageFolderVideoDataset

### ISSUE THREAD

**Issue by gngdb** [torchvideo/torchvideo#62]
Raising `AttributeError` when trying to apply `NormalizeVideo` transform. Notebook showing minimal reproducible code and output is here: https://gist.github.com/gngdb/3101d6078f711b8d2e3fc2f77da33390

Or, as a script:

```
import tempfile
import os
from PIL import Image
import numpy as np
from torchvideo import datasets, transforms
from pathlib import Path

def generate_image():
    x = np.uint8(np.random.rand(112, 112, 3)*255.)
    return Image.fromarray(x)

def generate_data(tempdir):
    for i in range(2):
        clip_dir = os.path.join(tempdir, f"clip{i}")
        os.mkdir(clip_dir)
        for t in range(1,5):
            im = generate_image()
            impath = os.path.join(tempdir, f"clip{i}", f"frame_{t:03d}.jpg")
            im.save(impath)

with tempfile.TemporaryDirectory() as tmpdirname:
    generate_data(tmpdirname)
    mean = [0.43216, 0.394666, 0.37645]
    std = [0.22803, 0.22145, 0.216989]
    transform = transforms.NormalizeVideo(mean, std)
    dataset = datasets.ImageFolderVideoDataset(tmpdirname, "frame_{:03d}.jpg", transform=transform)
    print(dataset[0])
```

**Comment by willprice** [comment]
Hi @gngdb,

I think you'll need to chain a few transforms to do what you're after

```python
import torchvideo.transforms as VT
...
transform = VT.compose([
  VT.CollectFrame(),
  VT.PILVideoToTensor(),
  VT.NormalizeVideo(mean, std)
])
...
dataset = datasets.ImageFolderVideoDataset(tmpdirname, "frame_{:03d}.jpg", transform=transform)
```

By default the datasets yield an iterator of PIL videos. The type of the `transform` arg is `Optional[Callable[[Iterator[Image]], Tensor]]` which means that the transform takes a single arg of type `Iterator[Image]` and must produce a `Tensor` as output.

**Comment by gngdb** [comment]
Ah I got confused because when I queried without a transform argument I would get a Tensor as output, so I assumed that is what would be passed to `NormalizeVideo`. I see now `PILVideoToTensor` is [applied as the default](https://github.com/torchvideo/torchvideo/blob/master/src/torchvideo/datasets/image_folder_video_dataset.py#L75) if `transform=None`.
